experiments/experiment.go

```diff
@@ -290,6 +290,7 @@ func (ec *experimentContext) createTemplateService(template *v1alpha1.TemplateSp
 	for _, ctr := range rs.Spec.Template.Spec.Containers {
 		for _, port := range ctr.Ports {
 			servicePort := corev1.ServicePort{
+				Name:       port.Name,
```

Comment:
I believe ports can also be unnamed. what is the default value here, is it compatible? (ideally type's default which is "")

Reply:
[Documentation](https://pkg.go.dev/k8s.io/api@v0.26.1/core/v1#ContainerPort) says it will be omitted if the name is not given. Should we make it by default an empty string if not given?

Reply:
I would not set it at all